docs/DeveloperGuide.adoc

```diff
@@ -349,6 +398,10 @@ _{More to be added}_
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `11` or above installed.
 .  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+.  Should be able to display student's photos in 4KKKKKK resolutionnnnn
+.  Application should respond within 5 seconds
+.  Should have a clean and minimalist interface
```

Comment:
Clean and minimalist interface may not be well defined (it's hard to determine when an interface is clean or minimalist enough)